build.py

```diff
@@ -68,6 +68,13 @@
     help="Build and run the integration tests (default is --no-integration-tests)",
 )
 
+parser.add_argument(
```

Comment:
Added this for the watch.mjs script to use, so it can just build the web version for speed (else it will run the nodejs wasm build also)